packages/settings-library/src/settings_library/storage.py

```diff
@@ -15,11 +15,20 @@ class StorageSettings(BaseCustomSettings, MixinServiceSettings):
     STORAGE_PORT: PortInt = DEFAULT_AIOHTTP_PORT
     STORAGE_VTAG: VersionTag = parse_obj_as(VersionTag, "v0")
 
+    @cached_property
+    def base_url(self) -> str:
+        # e.g. http://storage:8000
+        return self._compose_url(
+            prefix="STORAGE",
+            port=URLPart.REQUIRED,
+            vtag=URLPart.EXCLUDE,
+        )
+
     @cached_property
     def api_base_url(self) -> str:
-        # http://storage:8080/v0
+        # e.g. http://storage:8080/v0
         return self._compose_url(
-            prefix="storage",
+            prefix="STORAGE",
```

Comment:
are you sure about the upper case here? it does fit with the example and with what we actually use no?

Reply:
Yep. see https://github.com/ITISFoundation/osparc-simcore/blob/e6267407a754fe66b50afb18d56eedf17d8634d4/packages/settings-library/src/settings_library/utils_service.py#L94-L95